scripts/zones/Selbina/IDs.lua

```diff
@@ -23,6 +23,8 @@ zones[xi.zone.SELBINA] =
         FISHING_MESSAGE_OFFSET        = 6565, -- You can't fish here.
         NOMAD_MOOGLE_DIALOG           = 6665, -- I'm a traveling moogle, kupo. I help adventurers in the Outlands access items they have stored in a Mog House elsewhere, kupo.
         SUBJOB_UNLOCKED               = 6868, -- You can now designate a support job.
+        FERRY_DEPARTING               = 7019, -- All aboard!≺Prompt≻
```

Comment:
Update extractor says this should be only `All aboard!` with an ID of `7016`

Reply:
Where can i find the update extractor? I got this from POLUtils?

Reply:
https://github.com/LandSandBoat/UpdateExtractor

just make sure to update your client if you haven't already, there was a patch a few days ago. The comments that go in the lines there need to match update extractor's output, not polutils

Reply:
removed the <prompt>

Reply:
it shows as 7019 on Octobers update 

  </thing>
  <thing type="DialogTableEntry">
    <field name="index">7019</field>
    <field name="text">All aboard!≺Prompt≻
</field>

Reply:
One sec, my polutils agrees. I think update extractor might've used some old cached files.

Reply:
sure let me know.

About the <prompt> part. should that be removed anyway?

Reply:
Yup, it looks like it was using stale files. New run agrees with this PR. And yes the `<Prompt>` should be removed

Reply:
already removed =) both of them